modules/openapi-generator/src/main/resources/JavaSpring/homeController.mustache

```diff
@@ -42,45 +23,19 @@ import static org.springframework.web.reactive.function.server.RouterFunctions.r
 @Controller
 public class HomeController {
 
-{{^useSpringfox}}
-    private static YAMLMapper yamlMapper = new YAMLMapper();
-
-    @Value("classpath:/openapi.yaml")
-    private Resource openapi;
-
-    @Bean
-    public String openapiContent() throws IOException {
-        try(InputStream is = openapi.getInputStream()) {
-            return StreamUtils.copyToString(is, Charset.defaultCharset());
-        }
-    }
-
-    @GetMapping(value = "/openapi.yaml", produces = "application/vnd.oai.openapi")
-    @ResponseBody
-    public String openapiYaml() throws IOException {
-        return openapiContent();
-    }
-
-    @GetMapping(value = "/openapi.json", produces = "application/json")
-    @ResponseBody
-    public Object openapiJson() throws IOException {
-        return yamlMapper.readValue(openapiContent(), Object.class);
-    }
-
-{{/useSpringfox}}
 {{#reactive}}
     @Bean
     RouterFunction<ServerResponse> index() {
         return route(
             GET("/"),
-            req -> ServerResponse.temporaryRedirect(URI.create("{{#useSpringfox}}swagger-ui.html{{/useSpringfox}}{{^useSpringfox}}swagger-ui/index.html?url=../openapi.json{{/useSpringfox}}")).build()
```

Comment:
Springdoc, like Springfox, should be optional. The API-first option shall deactivate Springdoc doc gen (and the raw OAS be served instead)